homeassistant/components/overkiz/number.py

```diff
@@ -48,6 +48,34 @@ class OverkizNumberDescription(NumberEntityDescription, OverkizNumberDescription
         max_value=4,
         entity_category=EntityCategory.CONFIG,
     ),
+    # SomfyHeatingTemperatureInterface
+    OverkizNumberDescription(
+        key=OverkizState.CORE_ECO_ROOM_TEMPERATURE,
+        name="Eco Room Temperature",
+        icon="mdi:thermometer",
+        command=OverkizCommand.SET_ECO_TEMPERATURE,
+        min_value=6,
+        max_value=29,
+        entity_category=EntityCategory.CONFIG,
+    ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_COMFORT_ROOM_TEMPERATURE,
+        name="Comfort Room Temperature",
+        icon="mdi:home-thermometer-outline",
+        command=OverkizCommand.SET_COMFORT_TEMPERATURE,
+        min_value=7,
+        max_value=30,
+        entity_category=EntityCategory.CONFIG,
+    ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_SECURED_POSITION_TEMPERATURE,
+        name="Freeze Protection Temperature",
+        icon="mdi:sun-thermometer-outline",
+        command=OverkizCommand.SET_SECURED_POSITION_TEMPERATURE,
+        min_value=5,
+        max_value=15,
+        entity_category=EntityCategory.CONFIG,
+    ),
```

Comment:
Also implemented by (upcoming) climate entity, however we need these number entities for device config as well. This device can be configured to auto mode and then it will switch between those temperatures based on a time schedule. 

In the climate entity, they can only change a single temperature, when that mode is active. These number entities allow an user to set the temperature per mode in advance.